vochain/state/process.go

```diff
@@ -161,6 +164,52 @@ func (v *State) CountProcesses(committed bool) (uint64, error) {
 	return count, nil
 }
 
+func (v *State) RegisterStartBlock(pid []byte, startBlock uint32) error {
+	v.Tx.Lock()
+	defer v.Tx.Unlock()
+	processesTree, err := v.Tx.DeepSubTree(StateTreeCfg(TreeProcess))
+	if err != nil {
+		return err
+	}
+	newStartBlock := []byte{}
+	binary.LittleEndian.PutUint32(newStartBlock, startBlock)
+	return processesTree.NoState().Set(pid, newStartBlock)
+}
+
+func (v *State) DeleteStartBlock(pid []byte) error {
+	v.Tx.Lock()
+	defer v.Tx.Unlock()
+	processesTree, err := v.Tx.DeepSubTree(StateTreeCfg(TreeProcess))
+	if err != nil {
+		return err
+	}
+	return processesTree.NoState().Delete(pid)
+}
+
+// MinOnGoingStartBlock returns the minimun start block of the current on going
```

Comment:
`ongoing` is a word, not two.
so this would be MinOngoingStartBlock

also, `minimum` is the correct spelling

Reply:
Sorry about my english ;(